<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightPanel</class>
 <widget class="QWidget" name="RightPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="right_tab">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>9</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="resource_tab">
    <attribute name="title">
     <string>Resource</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="renderer_tab">
    <attribute name="title">
     <string>Renderer</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="camera_tab">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
